modules/rosa-sts-about-iam-resources.adoc

```diff
@@ -0,0 +1,87 @@
+// Module included in the following assemblies:
+//
+// * rosa_getting_started_sts/rosa-sts-creating-cluster.adoc
+
+[id="rosa-sts-about-iam-resources_{context}"]
+= About IAM resources for clusters that use STS
+
+To deploy a {product-title} (ROSA) cluster that uses STS, you must create the following AWS Identity Access Management (IAM) resources:
+
+* Specific account-wide AWS IAM roles and policies that provide the STS permissions required for ROSA support, installation, control plane and compute functionality.
+* Cluster-specific Operator roles and policies that permit the ROSA cluster Operators to carry out core OpenShift functionality.
```

Comment:
Just roles, not policies (those are account-wide)

Reply:
This is now resolved.